src/queryExecutionContext/EndpointComponent/AggregateEndpointComponent.ts

```diff
@@ -50,50 +51,43 @@ export class AggregateEndpointComponent implements ExecutionContext {
     this.aggregateValues = [];
     this.aggregateValuesIndex = -1;
 
-    try {
-      const { result: resources, headers } = await this._getQueryResults();
+    const { result: resources, headers } = await this._getQueryResults();
 
-      resources.forEach((resource: any) => {
-        // TODO: any
-        this.localAggregators.forEach(aggregator => {
-          let itemValue;
-          // Get the value of the first property if it exists
-          if (resource && Object.keys(resource).length > 0) {
-            const key = Object.keys(resource)[0];
-            itemValue = resource[key];
-          }
-          aggregator.aggregate(itemValue);
-        });
-      });
-
-      // Get the aggregated results
+    resources.forEach((resource: any) => {
+      // TODO: any
       this.localAggregators.forEach(aggregator => {
-        this.aggregateValues.push(aggregator.getResult());
+        let itemValue;
+        // Get the value of the first property if it exists
```

Comment:
Unrelated to this change:

Can we create a work item to track aggregation pipeline using the higher precision aggregate result?

The changes related to this file will explain:

https://msdata.visualstudio.com/CosmosDB/_git/CosmosDB?path=%2FProduct%2FSDK%2F.net%2FMicrosoft.Azure.Documents.Client%2FQuery%2FAggregation%2FMinMaxAggregator.cs&version=GBmaster&_a=contents

Reply:
https://github.com/Azure/azure-cosmos-js/issues/340